pkg/helmexec/exec.go

```diff
@@ -484,7 +490,7 @@ func (helm *execer) exec(args []string, env map[string]string) ([]byte, error) {
 	}
 	cmd := fmt.Sprintf("exec: %s %s", helm.helmBinary, strings.Join(cmdargs, " "))
 	helm.logger.Debug(cmd)
-	outBytes, err := helm.runner.Execute(helm.helmBinary, cmdargs, env)
+	outBytes, err := helm.runner.Execute(helm.helmBinary, cmdargs, env, helm.enableLiveOutput)
```

Comment:
I believe there are some execer methods that needs the whole output to be returned to the caller like `execer.List`.

For example, the `List` method returns the whole helm-list output so that the caller can use it to decide if a helm release is already installed onto the cluster or not, which affects what `helmfile sync` and `helmfile apply` do when the release exists in the cluster but instructed to be uninstalled by setting `installed: false` like

```
releases:
- name: myapp
  # This instructs Helmfile to uninstall the release by running `helm uninstall myapp` if `helm list` output showed the `myapp` release as installed
  installed: false
```

Perhaps we'd need to make the enableLiveOutput flag a parameter to the `execer.exec` function, so that each caller `execer` function can decide which value to `exec`, either `helm.enableLiveOutput` or a hard-coded `false`? 🤔

Reply:
thanks again for the heads up @mumoshu, I have followed your recommendation and added the ability of the caller to override the `enableLiveOutput` when calling `execer.exec` by adding a new pointer parameter, callers that want to explicitly disable live output will pass it as a reference for a hard-coded `false`, other callers that don't care about it, will pass it as a `nil`, and then `helm.enableLiveOutput` will be used by default inside the `execer.exec` function

Reply:
Hi @mumoshu, sorry for the direct mention here, but I just want to check your opinion about the final implementation in this PR, do you believe is it good enough? do you suggest any change or further tests? thanks in advance.